tests/ui/privacy/where-priv-type.stderr

```diff
@@ -8,59 +8,137 @@ LL | pub struct S
    = note: for more information, see issue #34537 <https://github.com/rust-lang/rust/issues/34537>
    = note: `#[warn(private_in_public)]` on by default
 
+warning: type `PrivTy` is more private than the item `S`
+  --> $DIR/where-priv-type.rs:25:1
+   |
+LL | struct PrivTy;
+   | ------------- but type `PrivTy` is only usable at visibility `pub(crate)`
+...
+LL | pub struct S
+   | ^^^^^^^^^^^^ struct `S` is reachable at visibility `pub`
```

Comment:
Same issue with the order of labels as in https://github.com/rust-lang/rust/pull/112413.
Let's convert these labels to notes for now.